labman/gui/templates/quantification.html

```diff
@@ -28,33 +24,36 @@
     }
   };
 
+  /**
+   * Determine an appropriate clipping value for the heatmap's maximum value.
+   *
+   * @param {Float} plateType String describing the plate type.
+   * @returns A maximum expected value for the colorscale of a heatmap.
+   *
+   */
+  function clippingForPlateType(plateType) {
+    if (plateType == '16S library prep') {
```

Comment:
These should be updated to include:

`['gDNA', 'compressed gDNA']`

with default clipping at 20